### DISCUSSION TITLE
Reed study spectrums always show constant loop-gain

### DISCUSSION THREAD

**Discussion by Sonoton** [edwardkort/WWIDesigner#136] [Q&A] [answered]
In the reed studies, the spectrum plot of the loop gain shows a uniformly high gain (no resonances, except that the black impedance ratio line has point where it passes through zero (ie, "resonances").  Is there a reason the loop gain is shown as constant over all frequencies?  There are no numerical values printed out in the console output.
Thank you for your time.
![FlatLoopGain](https://user-images.githubusercontent.com/71807472/134826315-f6c8fc41-a6eb-42d6-ae27-36e888481101.gif)

**Comment by bhp1** [comment, marked as answer]
For whistles and flutes, we use an algorithm for calculating loop gain from:

Roman Auvray, Benoit Fabre, Pierre-Yves Lagree, "Regime change and oscillation thresholds in recorder-like instruments", _J. Acoust. Soc. Am._ 131 (2), February 2012.

We don't have a comparable algorithm for loop gain in reed mouthpieces.  (If you know of one, please let us know.)  Without one, we arbitrarily assign a constant loop gain of 1.

**Reply by Sonoton** [reply]
Thank you for your patience.  The loop gain calculation at the Barkhausen/Nyquist point (Eq 20, and just using Eqs 2 ,3,and 4) in the Auvray paper is straightforward for the linear case, and the nonlinear methodology is also provided.  A similar paper which describes the linear and nonlinear cases, for jets and reeded instruments is Fletcher 1979.  The plots in Fletcher indicate that the same methodology may be considered to be applicable.  Best Regards, and thank you again.
[Fletcher1979.pdf](https://github.com/edwardkort/WWIDesigner/files/7233352/Fletcher1979.pdf)
